universal-application-tool-0.0.1/test/BrowserTest.java

```diff
@@ -31,24 +35,49 @@ protected TestBrowser provideBrowser(int port) {
   }
 
   @Test
-  public void test() {
+  public void homePage() {
     browser.goTo("http://localhost:" + play.api.test.Helpers.testServerPort());
```

Comment:
nit: the hard-coded string can be replaced with `LOCALHOST` now

Reply:
Actually the whole arg can be replaced with `BASE_URL`

Reply:
Thanks for catching that, it was my reason for making the string in the first place